src/ontology/uberon-edit.obo

```diff
@@ -224018,6 +224018,23 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 property_value: curator_notes "The perigastric lymph nodes include at least the right paracardial lymph nodes, left paracardial lymph nodes, lesser curvature lymph nodes, greater curvature lymph nodes, suprapyloric lymph nodes and infrapyloric lymph nodes. PMID:26973384" xsd:string
 property_value: dcterms-date "2023-05-11T12:28:12Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480060
+name: paraspinal region
+def: "The region of the body adjacent to the vertebral column." [NCIT:C166000]
+synonym: "paraspinal" EXACT [NCIT:C129461]
+is_a: UBERON:0005174 ! dorsal region element
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date "2023-05-16T06:37:51Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480061
+name: pericardial region
+def: "The region of the body adjacent to the pericardium and heart." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0005177 ! trunk region element
```

Comment:
Same concern as with the previous term. [Trunk region element](http://purl.obolibrary.org/obo/UBERON_0005177) is defined as an [organ](http://purl.obolibrary.org/obo/UBERON_0000062). I’d suggest [subdivision of trunk](http://purl.obolibrary.org/obo/UBERON_0009569) as a parent.